frontend/src/pages/auth/signin/SignIn.tsx

```diff
@@ -1,11 +1,50 @@
 import React from 'react';
-import {Titles} from "../../../constansts/titles";
+import {Box, Button} from "@mui/material";
+import "../../../common/styles/button.css";
+import {useAppDispatch} from "../../../stores/redux/hooks";
+import {signIn} from "../../../stores/redux/loggedUser/loggedUserSlice";
+import {getUserService} from "../../../services/user";
+import {Buttons} from "../../../common/constansts/buttons";
+import {logInService} from "../../../services/auth";
+import {User} from "../../../types/user";
+import {UserType} from "../../../types/userType";
+import {ILoggedUser} from "../../../types/loggedUser";
+import {decodeToken} from "../../../common/utils";
 
 const SignIn: React.FC = () => {
+    const dispatch = useAppDispatch()
+
+    const createLoggedUser = (token: string, user: User): ILoggedUser => {
+        return {
+            id: user.id,
+            email: user.email,
+            name: user.name,
+            userType: UserType.Customer,
+            token: token
```

Comment:
```suggestion
            token,
```
If key and value variable names are same, you can pass only value